### ISSUE TITLE
"west zephyr-export" does not create cmake directories and errors out.

### ISSUE THREAD

**Issue by issmirnov** [zmkfirmware/zmk#1030]
TLDR: `west zephyr-export` fails due to cmake not having appropriate parent folders set in `~/.cmake/packages/*`

## Background

I'm following https://zmk.dev/docs/development/setup, doing a native Linux setup. 

After cloning the repo and installing pip, I'm running `west zephyr-export` to set up the environment. I get fatal errors such as this:

```
FATAL ERROR: command exited with status 1: /usr/bin/cmake -P ~/Projects/zmk-keeb/zmk/zephyr/share/zephyrunittest-package/cmake/zephyr_export.cmake
```

After manually running the command, we can see the underlying issue:

```
$ /usr/bin/cmake -P ~/Projects/zmk-keeb/zmk/zephyr/share/zephyr-package/cmake/zephyr_export.cmake
CMake Error at zephyr/share/zephyr-package/cmake/zephyr_export.cmake:26 (file):
  file failed to open for writing (No such file or directory):

    /home/vania/.cmake/packages/Zephyr/5577177c735b0012cdbf97d48b260e9a
```

## Workaround

After manually creating the `/home/vania/.cmake/packages/Zephyr` (and for some reason needing a quick `chown -R $user` on it) the cmake command succeeds. 

The same error then happens for the other cmake package (`ZephyrUnittest`), requiring manual intervention.

After these mitigation steps are taken, the `west zephyr-export` command succeeds

## Recommendation

Either the `west` command or the `cmake` wrappers should check for the existence of those folders and create them on behalf of the user.

**Comment by nmunnich** [comment]
This is a Zephyr issue and not a ZMK one. I would hope that this has been resolved in more recent Zephyr versions.
